docs/appendices/release-notes/5.7.0.rst

```diff
@@ -85,6 +85,10 @@ None
 Administration and Operations
 -----------------------------
 
+- Added support for
+  :ref:`Google Cloud Storage repositories <sql-create-repo-gcs>` to write and
+  restore snapshots on Google Cloud Storage.
```

Comment:
We wanted to have a external contributor section right ?

Reply:
Yes